src/www/ui_components/server-card.html

```diff
@@ -155,7 +159,7 @@
         ></server-connection-viz>
         <div id="serverInfo">
           <div id="serverName">[[serverName]]</div>
-          <div id="serverHost">[[serverHost]]:[[serverPort]]</div>
+          <div id="serverHost">[[serverHost]]</div>
```

Comment:
Perhaps we should still show server and port, but only while connected?
This is really a UI question.

/cc @cjhenck